pages/linux/snap.md

```diff
@@ -0,0 +1,24 @@
+# snap
+
+> Tool for managing the *snap* self-contained softwawre packages.
+> Similar to what *apt* is for *.deb*.
+
+- Look for an app:
+
+`snap find {{package}}`
+
+- Install software:
```

Comment:
`software` seems a bit vague. Perhaps `a package` would be better?

Reply:
or app, to be consistent with the previous entry.

Reply:
I like "software" because snap doesn't necessarily carries "applications" in the mainstream sense of the word. Snap can also be only a library or some other kind of pile of code.

I didn't used package, because newbies not always know what "package" actually means. But yeah "package" could do since it can be used in an equally broad sense as "software".

Reply:
It's a word that's used by both `apt` and `npm`, so I thought it might be best to maintain consistency.